js_modules/dagster-ui/packages/ui-core/src/runs/RunRoot.tsx

```diff
@@ -166,6 +173,30 @@ export const RunRoot = () => {
 // eslint-disable-next-line import/no-default-export
 export default RunRoot;
 
+const RunAssetTags = (props: {run: RunFragment}) => {
+  const {run} = props;
+  const {data, loading} = useQuery<RunAssetsQuery, RunAssetsQueryVariables>(RUN_ASSETS_QUERY, {
+    variables: {runId: run.id},
+    skip: isHiddenAssetGroupJob(run.pipelineName),
+  });
+
+  if (isHiddenAssetGroupJob(run.pipelineName)) {
+    return <AssetKeyTagCollection useTags assetKeys={assetKeysForRun(run)} />;
+  }
+
+  if (loading) {
+    return <div style={{color: Colors.textLight()}}>Loading…</div>;
+  }
+
+  if (!data || data.pipelineRunOrError.__typename !== 'Run') {
+    return null;
+  }
```

Comment:
It seems like it could be a little awkward to show `Loading…` among a list of tags in the Run header, and also maybe kind of odd if it disappears in the empty case. Maybe just don't show any loading state?